raisinbread/RB_files/RB_permissions.sql

```diff
@@ -64,5 +64,7 @@ INSERT INTO `permissions` (`permID`, `code`, `description`, `moduleID`, `action`
 INSERT INTO `permissions` (`permID`, `code`, `description`, `moduleID`, `action`, `categoryID`) VALUES (70,'api_docs','API Documentation: View LORIS API Manual',NULL,NULL,2);
 INSERT INTO `permissions` (`permID`, `code`, `description`, `moduleID`, `action`, `categoryID`) VALUES (71,'electrophysiology_browser_edit_annotations','Annotations',41,'Create/Edit',2);
 INSERT INTO `permissions` (`permID`, `code`, `description`, `moduleID`, `action`, `categoryID`) VALUES (72,'monitor_eeg_uploads','Monitor EEG uploads',47,'Create/Edit',2);
+INSERT INTO `permissions` (`permID`, `code`, `description`, `moduleID`, `action`, `categoryID`) VALUES (73,'conflict_resolver','Parameter Type Descriptions',46,NULL,2);
```

Comment:
"conflict_resolver" is the "UNIQUE KEY" so you can not insert it twice, the test will stuck here. 
"UPDATE permissions
SET moduleID = (select ID FROM modules WHERE Name = 'dictionary') 
WHERE moduleID IN (SELECT ID FROM modules WHERE Name = 'datadict');"

Reply:
yes, you can keep the sql patch. 
you need to reformat the code to fix the error.
FILE: .../Loris/modules/conflict_resolver/php/conflict_resolver.class.inc
----------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
----------------------------------------------------------------------
 41 | WARNING | Line exceeds 85 characters; contains 122 characters
----------------------------------------------------------------------


FILE: ...s/Loris/modules/conflict_resolver/test/conflict_resolverTest.php
----------------------------------------------------------------------
FOUND 0 ERRORS AND 3 WARNINGS AFFECTING 3 LINES
----------------------------------------------------------------------
 100 | WARNING | Line exceeds 85 characters; contains 88 characters
 101 | WARNING | Line exceeds 85 characters; contains 96 characters
 102 | WARNING | Line exceeds 85 characters; contains 100 characters
----------------------------------------------------------------------

Reply:
ok I understand, and will do! I didn't know warnings would trigger these errors